apis/r/R/SOMASparseNDArray.R

```diff
@@ -189,6 +189,40 @@ SOMASparseNDArray <- R6::R6Class(
     #' @return A scalar with the number of non-zero elements
     nnz = function() {
       nnz(self$uri, config=as.character(tiledb::config(self$tiledbsoma_ctx$context())))
+    },
+
+    #' @description Increases the shape of the array as specfied. Raises an error
+    #' if the new shape is less than the current shape in any dimension. Raises
+    #' an error if the new shape exceeds maxshape in any dimension. Raises an
+    #' error if the array doesn't already have a shape: in that case please call
+    #' tiledbsoma_upgrade_shape.
+    #' @return No return value
+    resize = function(new_shape) {
```

Comment:
We need documentation for `new_shape`
```suggestion
    #' @description Increases the shape of the array as specfied. Raises an error
    #' if the new shape is less than the current shape in any dimension. Raises
    #' an error if the new shape exceeds maxshape in any dimension. Raises an
    #' error if the array doesn't already have a shape: in that case please call
    #' tiledbsoma_upgrade_shape.
    #' @param new_shape New shape to expand array to
    #' @return No return value
    resize = function(new_shape) {
```
With the new docs, `devtools::document()` needs to be run to update the Rd files in `man/`